locations/spiders/boots_no.py

```diff
@@ -8,7 +8,7 @@
 
 class BootsNOSpider(scrapy.Spider):
     name = "boots_no"
-    item_attributes = {"brand": "Boots", "brand_wikidata": "Q6123139"}
+    item_attributes = {"brand": "Apotek", "brand_wikidata": "Q4581428"}
```

Comment:
So, this is actually boots data. https://www.google.com/maps/@59.4847066,6.2513427,3a,75y,323.61h,87.33t/data=!3m6!1e1!3m4!1s2w1fIeJJUqLH34cxH5fpGw!2e0!7i16384!8i8192?entry=ttu

NSI needs to be updated to include Boots in NO.

Reply:
Cj is correct. I was to focused on the category and includes. Switched back and added Apply Category.
<details><summary>Stats</summary>

```python
{'atp/brand/Boots': 153,
 'atp/brand_wikidata/Q6123139': 153,
 'atp/category/amenity/pharmacy': 153,
 'atp/category/multiple': 153,
 'atp/field/city/missing': 153,
 'atp/field/email/missing': 153,
 'atp/field/image/missing': 153,
 'atp/field/opening_hours/missing': 153,
 'atp/field/phone/missing': 153,
 'atp/field/postcode/missing': 153,
 'atp/field/state/missing': 153,
 'atp/field/street_address/missing': 153,
 'atp/field/twitter/missing': 153,
 'atp/nsi/match_failed': 153,
 'downloader/request_bytes': 115978,
 'downloader/request_count': 309,
 'downloader/request_method_count/GET': 309,
 'downloader/response_bytes': 5131379,
 'downloader/response_count': 309,
 'downloader/response_status_count/200': 307,
 'downloader/response_status_count/404': 2,
 'elapsed_time_seconds': 99.406193,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2023, 10, 11, 20, 9, 55, 285049),
 'httpcompression/response_bytes': 37493501,
 'httpcompression/response_count': 307,
 'item_scraped_count': 153,
 'log_count/DEBUG': 473,
 'log_count/INFO': 10,
 'memusage/max': 289366016,
 'memusage/startup': 131891200,
 'request_depth_max': 2,
 'response_received_count': 309,
 'robotstxt/request_count': 2,
 'robotstxt/response_count': 2,
 'robotstxt/response_status_count/404': 2,
 'scheduler/dequeued': 307,
 'scheduler/dequeued/memory': 307,
 'scheduler/enqueued': 307,
 'scheduler/enqueued/memory': 307,
 'start_time': datetime.datetime(2023, 10, 11, 20, 8, 15, 878856)}
```
</details>